### ISSUE TITLE
Alphabetical sorting

### ISSUE THREAD

**Issue by jls2933** [digimezzo/dopamine-windows#264]
When sorting lists in alphabetical order, it seems that some words are ignored, like "The" for instance for group names.

Is it possible to maintain a list in configuration file of such words, or to add some other words depending of the language used ?

It would be great that words like "Les" in French, which have equivalent meaning of "The", were also ignored when sorting lists.

**Comment by Riajyuu** [comment]
> It would be great that words like "Les" in French, which have equivalent meaning of "The", were also ignored when sorting lists.

This can be hard as it requires languages comprehension. For programming it should be translation module or even neut-language network but neither is the point of Dopamine, an audio player.

**Comment by jls2933** [comment]
Yes, that's why a simple list of prefixes to strip off should be sufficient, and simple to implement. This list, initialized with actual english content, could be directly edited by people to add their own prefixes to be ignored.
